<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="ColorBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Label_connection">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Connection</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>581</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="gain_plot" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>330</y>
      <width>581</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="G1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>460</y>
      <width>16</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="G2">
    <property name="geometry">
     <rect>
      <x>151</x>
      <y>460</y>
      <width>16</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="G3">
    <property name="geometry">
     <rect>
      <x>202</x>
      <y>460</y>
      <width>16</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="G4">
    <property name="geometry">
     <rect>
      <x>253</x>
      <y>460</y>
      <width>16</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="G5">
    <property name="geometry">
     <rect>
      <x>304</x>
      <y>460</y>
      <width>16</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="G6">
    <property name="geometry">
     <rect>
      <x>355</x>
      <y>460</y>
      <width>16</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="G7">
    <property name="geometry">
     <rect>
      <x>406</x>
      <y>460</y>
      <width>16</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="G8">
    <property name="geometry">
     <rect>
      <x>457</x>
      <y>460</y>
      <width>16</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="G9">
    <property name="geometry">
     <rect>
      <x>508</x>
      <y>460</y>
      <width>16</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="G10">
    <property name="geometry">
     <rect>
      <x>559</x>
      <y>460</y>
      <width>16</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="G11">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>460</y>
      <width>16</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_linear_gain">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>480</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Linear gain</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_custom_gain">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>480</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Custom gain</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="GT1">
    <property name="geometry">
     <rect>
      <x>88</x>
      <y>530</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="GT2">
    <property name="geometry">
     <rect>
      <x>139</x>
      <y>530</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="GT3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>530</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="GT4">
    <property name="geometry">
     <rect>
      <x>241</x>
      <y>530</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="GT5">
    <property name="geometry">
     <rect>
      <x>292</x>
      <y>530</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="GT6">
    <property name="geometry">
     <rect>
      <x>343</x>
      <y>530</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="GT7">
    <property name="geometry">
     <rect>
      <x>394</x>
      <y>530</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="GT8">
    <property name="geometry">
     <rect>
      <x>445</x>
      <y>530</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="GT9">
    <property name="geometry">
     <rect>
      <x>497</x>
      <y>530</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="GT10">
    <property name="geometry">
     <rect>
      <x>547</x>
      <y>530</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="GT11">
    <property name="geometry">
     <rect>
      <x>598</x>
      <y>530</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>520</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset gain settings</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_set_gain">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Set gain</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_substract_offset">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>330</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Substract offset</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_offset">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>360</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Offset:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_offset">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>357</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_offset">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>390</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Offset calculus</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_freeze">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Freeze</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_auto">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>80</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto axis</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_fix">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>100</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Fix axis</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_ymax">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>140</y>
      <width>59</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>y max =</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ymax">
    <property name="geometry">
     <rect>
      <x>739</x>
      <y>136</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>1.0</string>
    </property>
    <property name="maxLength">
     <number>6</number>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_ymax">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>120</y>
      <width>20</width>
      <height>51</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_ymin">
    <property name="geometry">
     <rect>
      <x>811</x>
      <y>184</y>
      <width>20</width>
      <height>51</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="invertedAppearance">
     <bool>true</bool>
    </property>
    <property name="invertedControls">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_ymin">
    <property name="geometry">
     <rect>
      <x>681</x>
      <y>204</y>
      <width>59</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>y min =</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ymin">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>200</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>-1.0</string>
    </property>
    <property name="maxLength">
     <number>6</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>927</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionTCP_parameters"/>
    <addaction name="actionSettings_of_the_image"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionTCP_parameters">
   <property name="text">
    <string>TCP parameters</string>
   </property>
  </action>
  <action name="actionSettings_of_the_image">
   <property name="text">
    <string>Settings of the image</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>indent()</slot>
 </slots>
</ui>
